components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  pathname,
+  locale,
+}) {
+  const urlPrefix = fallbackLocale === locale ? '' : `${locale}.`
+  const image = `https://${urlPrefix}glamorous.rocks/static/images/banner.png`
```

Comment:
There's `getInitialLocaleProps` method that returns `locale` (uses `getLocaleAndHost` which return `locale` and `host`) and was wondering if using it would be acceptable to get the `BASE_URL`? Instead of returning `locale`, probably return `host` as well?